content/swiftui/concepts/views/terms/symmetricDifference.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'Text'
+Description: 'Returns a new set with the elements.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Buttons'
+  - 'Terms'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **symmetricDifference** in [SwiftUI](https://www.codecademy.com/resources/docs/swiftui) returns a new set with the elements that are either in the set or in the given
+sequence, but not in both.
+
+## Syntax
+```pseudo
+func symmetricDifference<S>(_other: S) -> Set<Element> where Element == S.Element, S:Sequence
```

Comment:
I think it's best to show this called on a set `mySet` rather than the way Apple Docs writes it out (it could be confusing to the reader if they're a beginner)